docs/references/incompatibility_matrix.md

```diff
@@ -0,0 +1,11 @@
+# Incompatibility matrix for Ubuntu Pro services
+
+Some Ubuntu Pro services are incompatible with each other. The following
+matrix display the services that are incompatible:
+
+|                 | fips    | fips-updates   | livepatch   | realtime-kernel   |    
```

Comment:
@s-makin I am not that happy with this table style, let me know if there another one that complies with the Ubuntu style we can use.

Reply:
I think it only looks a little odd because all of these services are incompatible with each other - this we could just as easily say as "you can't enable any of these services at the same time", which doesn't need a table. 

Looking at it now, I'm thinking it might make more sense if we change it to a "compatibility" (rather than incompatibility) matrix, and then also include the other services to show the full table of what you can and can't enable together? wdyt?

Reply:
I think that makes sense, let's see how the table will look like

Reply:
@s-makin I have made the compatibility matrix with all the services we support today. Let me know what you think